docs/reference/quickstart/getting-started.asciidoc

```diff
@@ -1,51 +1,81 @@
 [[getting-started]]
-== Quick start: Add data using Elasticsearch APIs
+== Quick start: Indices, documents, and mappings
 ++++
-<titleabbrev>Basics: Add data using APIs</titleabbrev>
+<titleabbrev>Basics: Indices, documents, and mappings</titleabbrev>
 ++++
 
 In this quick start guide, you'll learn how to do the following tasks:
 
-* Add a small, non-timestamped dataset to {es} using Elasticsearch REST APIs.
-* Run basic searches.
-
-[discrete]
-[[add-data]]
-=== Add data
-
-You add data to {es} as JSON objects called documents.
-{es} stores these
-documents in searchable indices.
-
-[discrete]
-[[add-single-document]]
-==== Add a single document
-
-Submit the following indexing request to add a single document to the
-`books` index.
-The request automatically creates the index.
+* Create an index
```

Comment:
should these be bookmark links to the section, Google can use lists like this to build cards on search that can help provide a card like experience that helps customers navigate

Reply:
can be totally, we have the right-side "On this page" that is automatically generated for us, so I'm not sure what the best practice is here

will be great to establish pattern for future in any case once we decide 👍

Reply:
replaced with prose in https://github.com/elastic/elasticsearch/pull/113558/commits/31d8006392b2eb3e578309b43493ad420c6a99e6